pkgs/development/python-modules/sphinx-fortran/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, future
+, numpy
+, sphinx
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "sphinx-fortran";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "VACUMM";
+    repo = pname;
+    rev = "${version}";
+    sha256 = "sha256-frep1YC8IjSq6r5K3iSYfg7aEmFuCiTJmE/BaPFCMt4=";
+  };
+
+  propagatedBuildInputs = [
+    future
+    numpy
+    sphinx
+    six
+  ];
+
+  pythonImportsCheck = [ "sphinxfortran" ];
+
+  # Test are failing: Sphinx configuration error
+  # checkInputs = [
+  #   pytestCheckHook
+  # ];
```

Comment:
The tests fail because they were written for python2. 
I tried applying `2to3` to the relevant file, but then the tests when trying to use deprecated functions and attributes.

Upstream seem to have updated several files to python3 since version `1.1.1`. As such I propose you package the newest unstable revision `394ae990b43ed43fcff8beb048632f5e99794264` as version `unstable-2022-03-02`.